evap/staff/views.py

```diff
@@ -1051,6 +1051,43 @@ def course_copy(request, course_id):
     )
 
 
+@require_POST
+@manager_required
+@transaction.atomic
+def create_exam_evaluation(request):
+    evaluation = get_object_from_dict_pk_entry_or_logged_40x(Evaluation, request.POST, "evaluation_id")
+    exam_date = request.POST.get("exam_date")
```

Comment:
This will cause a server error if the parameter is missing, right? It should be a 4XX (client) error, probably 400 bad request.

Reply:
I think l. 1058  was fine and @richardebeling meant l. 1059, right?

Reply:
Yup, `get_object_from_dict_pk_entry_or_logged_40x(Evaluation, request.POST, "evaluation_id")` is fine I think.

Keep in mind that `strptime` below will also throw errors, this should also be converted into a bad request.